src/main/java/com/novauc/SpringBasicsFRController.java

```diff
@@ -0,0 +1,88 @@
+package com.novauc;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.ui.ModelMap;
+import org.springframework.web.bind.annotation.*;
+
+import javax.servlet.http.HttpSession;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by souporman on 3/13/17.
+ */
+@Controller
+@RequestMapping("/")
+public class SpringBasicsFRController {
+
+    @Autowired
+    Messager userMessages;
+
+//    static ArrayList allMessages = new ArrayList();
+
+    @RequestMapping(path = "/", method = RequestMethod.GET)
+    public String home(Model model, HttpSession session) {
+        model.addAttribute("name", session.getAttribute("userName"));
+        for(Messages looper : userMessages.viewAllMessages()) {
+            model.addAttribute("messages",looper);
+        }
+        return "home";
+    }
+
+    @RequestMapping(value = "/login", method = RequestMethod.POST)
+    public String logout(Model model,HttpSession session, String userName){
+        session.setAttribute("userName",userName);
+        model.addAttribute("name", session.getAttribute("userName"));
+        return"redirect:/";
+    }
```

Comment:
Yea you made this more difficult than it should have been, but you probably learned a lot. Ability to learn and research is a very desirable skill.